library/src/main/java/com/schibsted/spain/barista/BaristaSwipeRefreshActions.java

```diff
@@ -0,0 +1,17 @@
+package com.schibsted.spain.barista;
+
+import android.support.annotation.IdRes;
+import com.schibsted.spain.barista.custom.CustomConstraintViewAction;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.swipeDown;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayingAtLeast;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+
+public class BaristaSwipeRefreshActions {
+
+  public static void pullToRefresh(@IdRes int swipeRefreshId) {
+    onView(withId(swipeRefreshId)).perform(
+        new CustomConstraintViewAction(swipeDown(), isDisplayingAtLeast(80)));
```

Comment:
As I understand, this `isDisplayingAtLeast` matchers if a little hack for the pull to refresh, right?

Maybe we could make that more explicit in the code.
What do you guys think about creating a custom action like `SwipeRefeshAction` where we encapsulate this hack? That way all the magic for the pull to refresh is together in one place.